frontend/public/kubevirt/components/modals/loader.jsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+
+const ESC_KEY = 27;
+
+/**
+ * Empty loader
+ * // TODO: add loading icon
+ */
+export class Loader extends React.Component {
```

Comment:
Just a DummyLoader. We can change it once we have an input from designers. For now it just traps users interaction and closes the Dialog and resource loading if interrupted.